platform/broadcom/sai.mk

```diff
@@ -1,9 +1,18 @@
-BRCM_SAI = libsaibcm_4.3.5.3-3_amd64.deb
-$(BRCM_SAI)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.3/202012/libsaibcm_4.3.5.3-3_amd64.deb?sv=2020-04-08&st=2022-03-16T21%3A31%3A49Z&se=2037-03-17T21%3A31%3A00Z&sr=b&sp=r&sig=6meWjaM%2FMyQRJIigjqs4xc7ZCGotP0S94np26kn9zls%3D"
-BRCM_SAI_DEV = libsaibcm-dev_4.3.5.3-3_amd64.deb
+LIBSAIBCM_VERSION = 4.3.5.3-4
+LIBSAIBCM_BRANCH_NAME = REL_4.3.5.2
+LIBSAIBCM_URL_PREFIX = "https://sonicstorage.blob.core.windows.net/public/sai/bcmsai/$(LIBSAIBCM_BRANCH_NAME)/$(LIBSAIBCM_VERSION)"
+
+BRCM_SAI = libsaibcm_$(LIBSAIBCM_VERSION)_amd64.deb
+$(BRCM_SAI)_URL = "$(LIBSAIBCM_URL_PREFIX)/$(BRCM_SAI)"
+BRCM_SAI_DEV = libsaibcm-dev_$(LIBSAIBCM_VERSION)_amd64.deb
 $(eval $(call add_derived_package,$(BRCM_SAI),$(BRCM_SAI_DEV)))
-$(BRCM_SAI_DEV)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.3/202012/libsaibcm-dev_4.3.5.3-3_amd64.deb?sv=2020-04-08&st=2022-03-16T21%3A33%3A39Z&se=2037-03-17T21%3A33%3A00Z&sr=b&sp=r&sig=n5nodstTKbsJLZDjhQYmpFB3rafcYPybsMaici4%2FwfE%3D"
+$(BRCM_SAI_DEV)_URL = "$(LIBSAIBCM_URL_PREFIX)/$(BRCM_SAI_DEV)"
+
+# SAI module for DNX Asic family
+BRCM_DNX_SAI = libsaibcm_dnx_$(LIBSAIBCM_VERSION)_amd64.deb
```

Comment:
i am not sure we want to make such change in 202012 branch.

Reply:
removed DNX part